src/main/java/fi/hsl/transitdata/omm/Main.java

```diff
@@ -25,17 +25,32 @@ public static void main(String[] args) {
         final ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor();
 
         try {
+            String cancellationsFromTime = ConfigUtils.getEnvOrThrow("CANCELLATIONS_FROM_TIME");
```

Comment:
config.getString('omm.cancellationsFromTime')?

omm {
    cancellationsFromTime="foo" // "foo" = transitdata, "bar" = transitlog
    cancellationsFromTime=${?CANCELLATIONS_FROM_TIME}
}

Reply:
Perhaps there's no need for this after all since it won't be read from the conf file but straight from the env variable (in Main). Clear error will be printed on how the env cancellationsFromTime should be set in case it's missing. This is exactly how the similar functionality is implemented in transitdata-pubtrans-source.

Reply:
Refactored the cancellationsFromTime -config.